.github/workflows/on-pullrequest.yml

```diff
@@ -0,0 +1,75 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+name: On Pull Request
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      # TODO: restore this with AA-1601, except run in a PS-specific
+      # workflow, so that the C# build and CodeQL do not run
+      # unnecessarily.
+      # - "**/*.ps1"
+      # - "**/*.psm1"
+      - "**/*.cs"
+      - "**/*.csproj"
+      - ".github/**/*.yml"
+  workflow_dispatch:
+
+jobs:
+  # TODO: restore this with AA-1601
+  # run-ps-lint:
+  #   name: PowerShell Linter
+  #   uses: Ed-Fi-Alliance-OSS/Ed-Fi-Actions/.github/workflows/powershell-analyzer.yml@main
```

Comment:
Please uncomment these lines.

In the future we might consider separating this pull request workflow into different workflows. For example there is no need to run the PowerShell linter - and spend time waiting for it to finish - if no PowerShell files were changed in the PR.

Reply:
Sounds like we are addressing this in the future

Reply:
Yes, although not under AA-1601 😄 .